src/hash/hash.c

```diff
@@ -2579,12 +2631,13 @@ static bool setGetHighEntropyValuesHeader(
 	detectionComponentState* state,
 	EvidenceKeyValuePair* pair) {
 	EXCEPTION_CREATE
-	TransformIterateResult result = TransformIterateGhevFromBase64
-	(pair->parsedValue,
-	 state->results->b.bufferTransform,
-	 state->results->b.bufferTransformLength,
-	 setSpecialHeadersCallback,
-	 state->evidence,exception);
+	TransformIterateResult result = TransformIterateGhevFromBase64(
```

Comment:
I think this routine should be called only if `IS_HEADER_MATCH(...)` returns true, otherwise it is called for every irrelevant header and value and tries to convert something that is not actually base64, and it's a performance penalty.